### ISSUE TITLE
i can play normal on nox app player but pokestops dont work

### ISSUE THREAD

**Issue by futi666** [finndev/PokeBuddy#983]
i can catch pokemons and fight in arenas but pokestops dont work in nox player and pokebuddy.
is this a ban or what?

**Comment by RoseCorpse** [comment]
same here, can catch and stuff but pokestops dont work

**Comment by shiitan5** [comment]
yes . i can play on my iphone and i can play using the hack but not the pokebuddy bot !

**Comment by Skolzack** [comment]
Same here and still more than 10hours...

**Comment by inkompetent** [comment]
#984

**Comment by AlexCtasy** [comment]
Well, 10hours and still softbanned, i guess we need wait like weeks to get unbanned or 24hs maybe xoaoa
